src/Appwrite/Network/Validator/Origin.php

```diff
@@ -122,11 +123,8 @@ public function isValid($origin): bool
             return true;
         }
 
-        if (\in_array($host, $this->clients)) {
-            return true;
-        }
-
-        return false;
+        $validator = new Hostname($this->clients);
+        return $validator->isValid($host);
```

Comment:
As a rule of thumb, I would always have a empty line before the return for more clarity.

```suggestion
        $validator = new Hostname($this->clients);

        return $validator->isValid($host);
```